sdk/storage/storage-blob/ChangeLog.md

```diff
@@ -1,54 +1,86 @@
 # Changelog
 
-* Improved comments for `BlockBlobURL.upload()`.
-* Exported `HttpRequestBody` type for who wants to implement a customized HTTP client.
```

Comment:
Are these two lines supposed to be deleted?

These came in along with the PR where I had migrated all the open PRs from azure-storage-js to the central repo.

Reply:
You can keep them since these are changes from last versino

---
In reply to: [298479732](https://github.com/Azure/azure-sdk-for-js/pull/4094#discussion_r298479732) [](ancestors = 298479732)

Reply:
Added them back with date `2019.01`